modules/ingestor/src/graph/vulnerability/mod.rs

```diff
@@ -94,6 +94,19 @@ impl Graph {
             .await?
             .map(|cve| VulnerabilityContext::new(self, cve)))
     }
+
+    pub async fn get_vulnerabilities<TX: AsRef<Transactional>>(
+        &self,
+        tx: TX,
+    ) -> Result<Vec<VulnerabilityContext>, Error> {
+        Ok(entity::vulnerability::Entity::find()
```

Comment:
Likewise, see `AdvisoryService` in `fundamental`. It's used by the advisory REST endpoints, and could likewise be consumed by GraphQL.

Reply:
Same as above